django/middleware/csrf.py

```diff
@@ -270,7 +270,7 @@ def process_view(self, request, callback, callback_args, callback_kwargs):
                     except DisallowedHost:
                         pass
 
-                # Create a list of all acceptable HTTP referers, including the
+                # Create a list of all acceptable HTTP referrers, including the
```

Comment:
Don't change the spelling of "referer" https://en.wikipedia.org/wiki/HTTP_referer.